expo-maps/ios/ExpoMaps/AppleMapsView.swift

```diff
@@ -26,5 +26,44 @@ public final class AppleMapsView: UIView {
       mapViewType = .standard
     }
     self.mapView.mapType = mapViewType
+    
   }
+
+  func setCompassButton(enable: Bool) -> Void {
+    self.mapView.showsCompass = enable
+  }
+    
+  func setMyLocationButton(enable: Bool) -> Void {
+    if (enable == true) {
+      self.enableMyLocationButton()
+    }
+  }
+    
+  func setLevelPickerButton(enable: Bool) -> Void {
+    //TODO: enable floor picker
+  }
+    
+  private func enableMyLocationButton() -> Void {
```

Comment:
```suggestion
  private func enableMyLocationButton() {
```
`Void` type can be omitted.